### ISSUE TITLE
show_topics method is not consistent across HdpModel and LdaModel

### ISSUE THREAD

**Issue by lucasoldaini** [piskvorky/gensim#389]
I've noticed that the show_topics method return topics in a different format depending on the model used when `formatted=False`. 

For example, HdpModel returns a list of list in the following format:

```python
>>> model = gensim.models.HdpModel(corpus=corpus, id2word=dictionary)
>>> topics = model.show_topics(-1, formatted=False)
>>> print topics
[
    [0,
        [
            (u'apple', 0.11383073569595759),
            (u'pie', 0.10697087154306592),
            (u'crust', 0.10290230477535894),
            …
        ]
    ],
…
 ]
```

while LdaModel does not return the topic number and specifies probabilities as first term of the tuples

```python
>>> model = gensim.models.LdaModel(corpus=corpus, id2word=dictionary)
>>> topics = model.show_topics(-1, formatted=False)
>>> print topics
[
    [
         (0.11383073569595759, u'apple'),
         (0.10697087154306592, u'pie'),
         (0.10290230477535894, u'crust'),
         …
    ],
…
 ]
```
(numbers and terms are fictitious)

Any reason for the discrepancies?

-Luca

**Comment by cscorley** [comment]
Indeed, this was brought up in #354. If you'd like to submit a PR, that'd be good :)

**Comment by lucasoldaini** [comment]
sure, I'll take care of it soon.

**Comment by tmylk** [comment]
Fixed in #448
